General/Settings.md

```diff
@@ -0,0 +1,292 @@
+
+Example:
+```Smalltalk
+domainValues: (UITheme allThemeClasses collect: [:c | c themeName -> c])
+```
+
+#### Launching a script
+Imagine that you want to launch an external configuration tool or that you want to allow one to configure the system or a particular package with the help of a script. In such a case you can declare a *launcher*. A launcher is shown with a label as a regular setting except that no value is to be entered for it. Instead, a button labelled *Launch* is integrated in the Settings Browser and clicking on it launch an associated script.
+
+Illustration: 
+![Launching a Script](./Settings_Image_LaunchingAScript.png)
+
+*More information in section 5.6 "Launching a script" of the [Deep Into Pharo book](http://books.pharo.org/deep-into-pharo/).*
+
+#### Start-up actions management
+
+Even if many preferences have been removed from Pharo because they were obsolete, there are a still a large number of them. And even if the Settings Browser is easy to use, it may be tedious to set up your own preferences even for a subset, each time you start working with a new image. A solution is to implement a script to set all your preferred choices. The best way is to create a specific class for that purpose. You can then include it in a package that you can reload each time you want to setup a fresh image. We call this kind of class a *Setting style*.  
+To manage *Setting styles*, the Settings Browser can be helpful in two ways. First, it can help you discover how to change a preference value, and second, it can create and update a particular style for you.
+
+*More information in section 5.7 "Start-up actions management" of the [Deep Into Pharo book](http://books.pharo.org/deep-into-pharo/).*
+
+#### Extending the Settings Browser
+
+The Settings Browser is by default able to manage simple preference types. These default possibilities are generally enough. But there are some situations where it can be very helpful to be able to handle more complex preference values.  
+As an example, let us focus on the text selection preferences. We have the primary selection and three other optional kinds of text  selection, the secondary selection, the ‘find and replace’ selection and the selection bar. For all selections, a background color can be set. For the primary, the secondaryand the find and replace selection, a text color can also be chosen.
+
+*More information in section 5.8 "Extending the Settings Browser" of the [Deep Into Pharo book](http://books.pharo.org/deep-into-pharo/).*
+
+## Credits
+Thomas Dupriez.  
+This page is based on chapter 5 of the [Deep Into Pharo book](http://books.pharo.org/deep-into-pharo/): The Settings Framework  
+<small><i><a href='http://ecotrust-canada.github.io/markdown-toc/'>Table of contents generated with markdown-toc</a></i></small>
```

Comment:
I think you can remove this last line, we never specify the tool we use to generate the toc. :-)